core/src/main/java/io/varhttp/parameterhandlers/VarWebSocketHandler.java

```diff
@@ -0,0 +1,35 @@
+package io.varhttp.parameterhandlers;
+
+import io.varhttp.RegisteredWebSockets;
+import io.varhttp.VarWebSocket;
+import io.varhttp.WebSocket;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import java.lang.reflect.Method;
+import java.lang.reflect.Parameter;
+
+@Singleton
+public class VarWebSocketHandler implements IParameterHandlerMatcher {
+    private RegisteredWebSockets registeredWebSockets;
+
+    @Inject
+    public VarWebSocketHandler(RegisteredWebSockets registeredWebSockets) {
+        this.registeredWebSockets = registeredWebSockets;
+    }
+
+    @Override
+    public int getPriority() {
+        return 10;
```

Comment:
a comment about why 10 is the right priority would serve to the readability quite a lot :-)

Reply:
It's dependent on the priorities of the other filters. Updating comments everywhere when changing priorities would not be very efficient. If the documentation does not detail what these are there for, or if the purpose of the priority method is not obvious, then the documentation is lacking.